src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/DBFilterBase.java

```diff
@@ -16,10 +16,7 @@
 import io.stargate.sgv2.jsonapi.service.shredding.model.DocumentId;
 import io.stargate.sgv2.jsonapi.util.JsonUtil;
 import java.math.BigDecimal;
-import java.util.Date;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
+import java.util.*;
```

Comment:
I was asked before by Ivan to keep the import statement expanded. Can we have explicit imports?

Reply:
FWTW, we as a team are free to decide whether to expand or not these imports. There's no universal consensus on "Right" approach.

I know some devs want fully expanded; others not. I think IDEs default to expanded for up to 4 from same packages, then wildcard if there are 5 or more imports.
This is a compromise between verbosity and visibility.

I can work with either approach.

Reply:
solved